Sources/StreamChat/Database/DatabaseContainer.swift

```diff
@@ -74,6 +74,8 @@ class DatabaseContainer: NSPersistentContainer {
     private let deletedMessageVisibility: ChatClientConfig.DeletedMessageVisibility?
     private let shouldShowShadowedMessages: Bool?
 
+    static var cachedModel: NSManagedObjectModel?
```

Comment:
This is never set to nil? Trying to understand what would be the impact of that.

Reply:
Yes this not set to `nil`, only initially is `nil`. We need to understand if we created it once or not